### DISCUSSION TITLE
Favorite and boost notifications of replies in v4.3.0 no longer show the content of the reply.

### DISCUSSION THREAD

**Discussion by LingGeekRain** [mastodon/mastodon#32323] [General]
Up until the most recent update, I could see at a glance in my notifications tab the content of the reply that was favorited or boosted. Now with v4.3.0 all it displays is who it's replying to.

Could you please fix it so that it once again shows the content of the reply so I can see at a glance and not need to click through?

**Comment by renchap** [comment]
Please do not open both a discussion and an issue for the same thing. As replied in the issue, this is already tracked here: #32196

**Reply by LingGeekRain** [reply]
Sorry. I saw an option to "create issue from discussion" and was trying to figure out how to do that.
